flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/MySqlSourceITCase.java

```diff
@@ -54,4 +55,26 @@ public void testConsumingAllEvents() throws Exception {
 
         env.execute("Print MySQL Snapshot + Binlog");
     }
+
+    @Test
+    public void testConsumingAllEventsWithJsonFormat() throws Exception {
```

Comment:
The ITCase is a little having and the test is set to `@ingore`, could you add a unit test to check all data types?

Reply:
I will refer to this `MySqlConnectorITCase#testAllTypes` to complete the ITCase.